Makefile.am

```diff
@@ -181,3 +182,12 @@ endif
 if ENABLE_MODULE_RECOVERY
 include src/modules/recovery/Makefile.am.include
 endif
+
+
+if ENABLE_RUST_NAIVETESTS
+$(ECC_SECP):
+	git clone https://github.com/elichai/ecc-secp256k1.git -b v0.1.0 2> /dev/null | true
+	cd ecc-secp256k1 && cargo build --release --features=ffi
+	cp ./ecc-secp256k1/target/release/libecc_secp256k1.a  .
+	cp ./ecc-secp256k1/ecc_secp256k1.h  ./src/
+endif
```

Comment:
This needs additional rules for `make clean`.

Reply:
fixed.
Do you think we should use `cargo clean` for cleanup or manually delete the `target/` dir?

Reply:
or should it just remove the whole `ecc-secp256k1` directory?